src/objects/Opportunity.object

```diff
@@ -844,6 +844,19 @@
         <type>LongTextArea</type>
         <visibleLines>3</visibleLines>
     </fields>
+    <fields>
+        <fullName>CommitmentId__c</fullName>
```

Comment:
@melissabarber have you had a chance to look at these fields and align with the rest of your efforts?